qml/StatisticsChartBox.qml

```diff
@@ -601,9 +601,16 @@ Rectangle {
     Dialog {
         id: infoDialog
         title: "Chart Interactive Controls"
-        standardButtons: Dialog.Ok
+        standardButtons: Dialog.Ok | Dialog.Help
         anchors.centerIn: Overlay.overlay
 
+        Component.onCompleted: {
+            standardButton(Dialog.Ok).text = qsTrId("Got it!")
+            standardButton(Dialog.Help).text = qsTrId("Extra Help...")
+        }
+
+        onHelpRequested: Qt.openUrlExternally("https://fast-dds-monitor.readthedocs.io/en/latest/rst/user_manual/chart_panel_index.html")
```

Comment:
```suggestion
        onHelpRequested: Qt.openUrlExternally("https://fast-dds-monitor.readthedocs.io/en/latest/rst/user_manual/chart_panel/chartbox.html#chart-interaction")
```

Reply:
This button should be and **extra** help, so I think it would be better to redirect the user to the chartbox section.